### ISSUE TITLE
Undefined array key

### ISSUE THREAD

**Issue by eduardokraus** [sokunthearithmakara/mod_interactivevideo#7]
When editing the video, it shows the error.
```Warning: Undefined array key "endscreentext" in /mod/interactivevideo/mod_form.php on line 447```

![Captura de Tela 2024-10-11 às 10 12 35](https://github.com/user-attachments/assets/e17c2012-e61c-4d9f-9485-d15116ca1330)

**Comment by sokunthearithmakara** [comment]
@eduardokraus I unfortunately couldn't reproduce this issue on my end. What version of Moodle are you testing on (looks like 4.1 or 4.2 based on your screenshot)? Also, it doesn't look like you're using a standard theme. May I know what theme you're using?

**Comment by eduardokraus** [comment]
4.1

**Comment by eduardokraus** [comment]
I created a YouTube video activity, and the "endtime" field was automatically set to "00:01:18.00". After saving and returning to the course, I clicked on "Edit Settings". When I tried to reopen the activity settings, an error occurred, which also appeared in Moodle 4.5.

![Captura de Tela 2024-10-11 às 16 16 33](https://github.com/user-attachments/assets/1ac18bf1-d025-4d29-b939-457b169f1eae)
![Captura de Tela 2024-10-11 às 16 17 13](https://github.com/user-attachments/assets/d9a96310-6eda-452a-abfa-e617410accae)
![Captura de Tela 2024-10-11 às 16 19 15](https://github.com/user-attachments/assets/90cb790a-439d-4629-aacf-1e5f4641d939)
